config/src/keyassignment.rs

```diff
@@ -501,13 +501,20 @@ fn default_fuzzy_description() -> String {
     "Fuzzy matching: ".to_string()
 }
 
+// FIXME: this is so ugly we should use `window::WindowLevel` instead
+#[derive(Debug, Clone, PartialEq, FromDynamic, ToDynamic)]
+pub enum SetWindowLevelLevel {
```

Comment:
I wasn't sure about adding the window crate here, just for an enum. Maybe we can move it somewhere else?

Reply:
Ah, currently `window` depends on `config`, so trying to reference a type from `window` here wouldn't be possible because it would introduce a dependency cycle.

It is OK to define `WindowLevel` here in config and use it in the `window` crate.